frontend/README.md

```diff
@@ -0,0 +1,10 @@
+# Setting up Dev Environment
+
+1. cd into frontend/dashboard
```

Comment:
You don't need thi step if the first thing is to run relay in relay-workflows-lib